### ISSUE TITLE
[QUESTION] Updating prescribed surface CH4 used in full-chemistry simulations

### ISSUE THREAD

**Issue by jimmielin** [geoschem/geos-chem#1626]
Hi GCST, @ltmurray,

@zsx-GitHub (Shixian Zhai) has pointed out in the ACMG AQ subgroup yesterday that the full-chemistry simulation in GEOS-Chem uses prescribed CH4 concentrations at the surface. According to the [GEOS-Chem Wiki](https://wiki.seas.harvard.edu/geos-chem/index.php/Tropospheric_chemistry_mechanism#CH4_concentrations) the mean surface CH4 concentrations prescribed are provided by @ltmurray based on 1983-2016 data:

> I have prepared spatially kriged (interpolated) monthly mean surface methane distributions from the NOAA GMD flask observations from the early 1980s through the present in a NetCDF file. This allows for seasonality in the methane concentrations and local maxima (e.g., south central US; western Russia) and minima. It is not appropriate to use these surface distributions fixed throughout the entire column (spatial heterogeneity will be reduced in the free troposphere), but I use them in all my current simulations to fix surface concentrations and advect methane at the cost of an additional transported tracer. Since the surface concentration is fixed, tropospheric chemistry equilibrates much faster than if we were to have online methane emissions, on the time-scale of tropospheric mixing. UCX does the same thing for its treatment of methane by default, but with a single mean surface value. Fixing surface concentrations and allowing advection is common treatment in other atmospheric chemistry models. 
> 
> It is now generally standard practice for global models to prescribe surface methane concentrations and allow it to advect. It doesn’t have large impacts on the short-lived chemistry, but my maps eliminate the artificial step gradients that the zonal bands impose and introduce realistic and dynamic spatial gradients in the troposphere. I think it is worth putting in tropchem as an advected tracer, but at the very least, we should use my mean meridional gradients to impose the fixed concentrations.
> 
> It’s no more work to maintain than the current treatment, which is constantly falling behind (and the UCX surface data is even further behind than tropchem).
> 
> The NetCDF methane files are ready and contain observations from Feb 1983 through Dec 2016, with extrapolated trends for 1979-2020.

@zsx-GitHub has pointed us to [Zhang et al. 2021](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2020GL089184) which quantified that 26.7% of the global O3 burden changes during 1980-2010 is driven by global methane increase. Given the recent increase in methane, it may make sense to update this input to present observations, since input from 2016-2020 are extrapolated trends and there is no data for 2020 and beyond as specified in `HEMCO_Config.rc`:

```
#==============================================================================
# --- NOAA GMD monthly mean surface CH4 ---
#==============================================================================
(((GMD_SFC_CH4
* NOAA_GMD_CH4 $ROOT/NOAA_GMD/v2018-01/monthly.gridded.surface.methane.1979-2020.1x1.nc SFC_CH4 1979-2020/1-12/1/0 RY xy ppbv * - 1 1
)))GMD_SFC_CH4
```

The [NOAA_GMD dataset from ExtData](http://geoschemdata.wustl.edu/ExtData/HEMCO/NOAA_GMD/v2018-01/) does not include a script to process these from inputs. @ltmurray, I was wondering if there are plans to update the dataset or potentially share the processing scripts to create updated monthly gridded surface CH4 concentrations from the observational data?

Thank you!

Best,
Haipeng

**Comment by ltmurray** [comment]
Yes, the updated files are slated for inclusion in the next version of the model. You can contact me offline in the meantime if you need them sooner.

**Comment by msulprizio** [comment]
@ltmurray @toddmooring The GCST still does not have this update, but we would like to include this in 14.2.0 if possible. Could one of you create a pull request at your earliest convenience?

**Comment by ltmurray** [comment]
@msulprizio the updated files are on my office computer, and I am in Nevada until next week for a GCAP meeting. Is early next week okay? @toddmorring has a version I shared last December, but he pointed out some issues with the data in the Canadian tar sands, so I have a newer version that does not include those observations.

**Comment by msulprizio** [comment]
We will aim to implement this update in 14.3.0 since 14.2.0 is now in the benchmarking phase.

**Comment by msulprizio** [comment]
Hi @ltmurray. Do you have an update on the availability of the updated CH4 boundary condition files?

**Comment by yantosca** [comment]
Closed by #1979

**Comment by msulprizio** [comment]
The pull request for this issue (#1979) has now been merged into the 14.4.0 development branch.
